CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Change log
+
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## Version [Unreleased] <sub><sup>now</sub></sup>
+
+## Version [0.3.2] <sub><sup>2022-02-02</sub></sup>
+
+### Fixed
+
+- Loosened active record support to include active record 6.1 #23
+- Fixed keyword arguments for ruby 3.0 support #21
```

Comment:
```suggestion
- Loosened active record support to include active record 6.1 [[#23](https://github.com/ManageIQ/activerecord-id_regions/pull/23)]
- Fixed keyword arguments for ruby 3.0 support [[#21](https://github.com/ManageIQ/activerecord-id_regions/pull/21)]
```

Reply:
No one else adds links to PR numbers and it adds a lot of noise for the file.
Thinking about just dropping all of this